test/TitleEscrowCreator.js

```diff
@@ -15,11 +15,12 @@ contract("TitleEscrowCreator", accounts => {
   let TitleEscrowCreatorInstance;
 
   before(async () => {
-    TitleEscrowCreatorInstance = await TitleEscrowCreator.new();
+    // TitleEscrowCreatorInstance = await ERC721.new();
```

Comment:
Can delete the entire block if its not running any code

Reply:
done. removed that block of code.